### PR TITLE
Add dos2unix

### PR DESCRIPTION
**Pull request by adbeda**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your
      recipe (see
      [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos)
      for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/dos2unix/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    - make prefix=$PREFIX install
+
+requirements:
+  build:
+    - {{ compiler("c") }}
+    - {{ compiler('cxx') }}
+
+  run:
+    - make
+    - libgcc
```
Comment (isuruf) [inline recipes/dos2unix/meta.yaml]: Remove this line

File: recipes/dos2unix/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    - make prefix=$PREFIX all
+    - make prefix=$PREFIX install
+
+requirements:
+  build:
+    - {{ compiler("c") }}
+    - {{ compiler('cxx') }}
+
+  run:
+    - make
```
Comment (isuruf) [inline recipes/dos2unix/meta.yaml]: This should be in build

Comment (adbeda) [reply recipes/dos2unix/meta.yaml]: got it

**Comment by adbeda** [comment]
@isuruf, before creating the recipe, i built this package on my mac with out any issue. Also, if I restrict the build to only osx, and skip both linux and windows, it works just fine. should i just limit the recipe to only Linux, at least for now?

File: recipes/dos2unix/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - folder: {{ pkg_src }}
+    url: https://{{ import_path }}/{{name}}-{{ version }}.tar.gz
+    sha256: bac765abdbd95cdd87a71989d4382c32cf3cbfeee2153f0086cb9cf18261048a
+
+build:
+  skip: True   # [win]
```
Comment (adbeda) [inline recipes/dos2unix/meta.yaml]: build for osx

**Comment by adbeda** [comment]
@sodre, is there a solution to deal with`libintl.h` for OSX ?

File: recipes/dos2unix/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,49 @@
+# based on:  /Users/cheng/Work/biobuilds/recipes/dos2unix/7.4.0
```
Comment (sodre) [inline recipes/dos2unix/meta.yaml]:
```suggestion
```

File: recipes/dos2unix/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+
```
Comment (sodre) [inline recipes/dos2unix/meta.yaml]:
```suggestion
```

File: recipes/dos2unix/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+{% set import_path = "waterlan.home.xs4all.nl/dos2unix" %}
```
Comment (sodre) [inline recipes/dos2unix/meta.yaml]:
```suggestion
{% set  name = "dos2unix" %}
```

File: recipes/dos2unix/meta.yaml [hunk 7]
```diff
@@ -0,0 +1,49 @@
...
+{% set version = "7.4.0" %}
+
```
Comment (sodre) [inline recipes/dos2unix/meta.yaml]:
```suggestion
```

File: recipes/dos2unix/meta.yaml [hunk 8]
```diff
@@ -0,0 +1,49 @@
...
+{% set name = "dos2unix" %}
```
Comment (sodre) [inline recipes/dos2unix/meta.yaml]:
```suggestion
```

File: recipes/dos2unix/meta.yaml [hunk 9]
```diff
@@ -0,0 +1,49 @@
...
+{% set pkg_src = (import_path).replace("/", os.sep) %}
```
Comment (sodre) [inline recipes/dos2unix/meta.yaml]:
```suggestion
```

File: recipes/dos2unix/meta.yaml [hunk 10]
```diff
@@ -0,0 +1,49 @@
...
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - folder: {{ pkg_src }}
```
Comment (sodre) [inline recipes/dos2unix/meta.yaml]:
```suggestion
```

File: recipes/dos2unix/meta.yaml [hunk 11]
```diff
@@ -0,0 +1,49 @@
...
+    url: https://{{ import_path }}/{{name}}-{{ version }}.tar.gz
```
Comment (sodre) [inline recipes/dos2unix/meta.yaml]:
```suggestion
    url: https://waterlan.home.xs4all.nl/{{name}}/{{name}}-{{ version }}.tar.gz
```

File: recipes/dos2unix/meta.yaml [hunk 12]
```diff
@@ -0,0 +1,49 @@
...
+    sha256: bac765abdbd95cdd87a71989d4382c32cf3cbfeee2153f0086cb9cf18261048a
+
+
+build:
+  skip: true  # [win]
+  number: 0
+  script:
+    - pushd {{ pkg_src }}
```
Comment (sodre) [inline recipes/dos2unix/meta.yaml]:
```suggestion
```

File: recipes/dos2unix/meta.yaml [hunk 13]
```diff
@@ -0,0 +1,49 @@
...
+    - make prefix=$PREFIX all
+    - make prefix=$PREFIX install
+
+requirements:
+  build:
+    - {{ compiler ('c') }}
+    - make
+    - cctools 895  # [osx]
```
Comment (sodre) [inline recipes/dos2unix/meta.yaml]:
```suggestion
```

File: recipes/dos2unix/meta.yaml [hunk 14]
```diff
@@ -0,0 +1,49 @@
...
+    - clang 4.0.1 1  # [osx]
```
Comment (sodre) [inline recipes/dos2unix/meta.yaml]:
```suggestion
```

File: recipes/dos2unix/meta.yaml [hunk 15]
```diff
@@ -0,0 +1,49 @@
...
+    - clang_osx-64 4.0.1 h1ce6c1d_11  # [osx]
```
Comment (sodre) [inline recipes/dos2unix/meta.yaml]:
```suggestion
```

File: recipes/dos2unix/meta.yaml [hunk 16]
```diff
@@ -0,0 +1,49 @@
...
+    - gettext   # [osx]
```
Comment (sodre) [inline recipes/dos2unix/meta.yaml]:
```suggestion
    - gettext
```

File: recipes/dos2unix/meta.yaml [hunk 17]
```diff
@@ -0,0 +1,49 @@
...
+test:
+  commands:
+    - dos2unix -h
+    - unix2dos -h
+
+about:
+  home: https://waterlan.home.xs4all.nl/dos2unix.html
+  license: BSD-2-Clause
+  license_family: BSD
+  license_file: {{ pkg_src }}/COPYING.txt
```
Comment (sodre) [inline recipes/dos2unix/meta.yaml]:
```suggestion
  license_file: COPYING.txt
```

File: recipes/dos2unix/meta.yaml (hunk 4 above)
Comment (adbeda) [reply recipes/dos2unix/meta.yaml]: I was just removing this

**Comment by sodre** [comment]
@adbeda,  you can skip dos2unix for Windows. It is already in the [msys2 channel](https://anaconda.org/msys2/m2-dos2unix)

**Pull request merged**